Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -340,8 +340,8 @@ ThrowCompletionOr<u64> to_temporal_date_time_rounding_increment(VM& vm, Object c
     return to_temporal_rounding_increment(vm, normalized_options, maximum, false);
 }
 
-// 13.14 ToSecondsStringPrecision ( normalizedOptions ), https://tc39.es/proposal-temporal/#sec-temporal-tosecondsstringprecision
-ThrowCompletionOr<SecondsStringPrecision> to_seconds_string_precision(VM& vm, Object const& normalized_options)
+// 13.14 ToSecondsStringPrecisionRecord ( normalizedOptions ), https://tc39.es/proposal-temporal/#sec-temporal-tosecondsstringprecision
```

Comment:
Please update the link here to: https://tc39.es/proposal-temporal/#sec-temporal-tosecondsstringprecisionrecord as well, otherwise LGTM

Reply:
Ah, sorry if I was misleading! We typically leave the Record name off the struct if there are no naming conflicts. I only meant to say to update the link itself

Reply:
Thanks for pointing this out!

I've update the link in my commit, and have renamed `SecondsStringPrecision` to `SecondsStringPrecisionRecord` so that we actually return the record the name mentions.

Reply:
> Ah, sorry if I was misleading! We typically leave the Record name off the struct if there are no naming conflicts. I only meant to say to update the link itself

Oh haha no worries, I'll revet the name of the struct, then

Reply:
Sorry, I just meant the record name, I can see how I was confusing in my edit. My bad!